### ISSUE TITLE
Use Cloudflare Turnstile in captcha module

### ISSUE THREAD

**Issue by air3ijai** [pk910/PoWFaucet#251]
As per documentation [Module: `captcha`](https://github.com/pk910/PoWFaucet/wiki/Operator-Wiki#module-captcha), faucet supports
 - [HCaptcha](https://dashboard.hcaptcha.com/sites)
 - [Google / reCaptcha](https://www.google.com/recaptcha/admin)
 
Also, there are [some closed issues](https://github.com/pk910/PoWFaucet/issues?q=turnstile), related to the [Cloudflare Turnstile](https://www.cloudflare.com/products/turnstile/). But I can't find anything related to it in the code or git log.

Is there a way to use Turnstile in the `captcha` module?

**Comment by pk910** [comment]
Good point :)

I'm currently using the turnstile captcha via a custom captcha implementation that basically allows randomizing the type of captcha for each visitor.
Turnstile is part of that "adapter script", but not actually supported by the faucet itself.

I'll gonna add proper support for the turnstile captcha when I have time - shouldn't be a big thing :)

**Comment by air3ijai** [comment]
Just switched to Cloudflare Turnstile, so no more click games - thanks!

[Module: `captcha`](https://github.com/pk910/PoWFaucet/wiki/Operator-Wiki#module-captcha) was updated and it would be usefully to update generated default config as well

[Use docker image](https://github.com/pk910/PoWFaucet/wiki/Operator-Wiki#use-docker-image)
```shell
docker run -u $UID:$GID -v $(pwd):/config pk910/powfaucet:v2.2.5 --datadir=/config --create-config
```

**Comment by air3ijai** [comment]
Btw, how we can add some text above the logo, like it is on the https://sepolia-faucet.pk910.de

<img width="994" alt="Screenshot 2024-08-23 at 15 30 55" src="https://github.com/user-attachments/assets/ce766194-1233-46b7-b9ce-1a8e4db0d58a">

**Comment by pk910** [comment]
These texts above the the logo are coming from the notification system.

This is currently barely documented.. Probably needs a section in the wiki..

The notifications are regularly read from an extra file, which allows pushing on the fly notifications without restarting or reloading the faucet. 
The filename is configurable via the config, but defaults to `faucet-status.yaml`:
```yaml
# faucet-config.yaml

faucetStatus:
    yaml: "./faucet-status.yaml"
```

This `faucet-status.yaml` looks as follows for the sepolia instance:

```yaml
- key: hashrate
  prio: 1
  text: |
    <span class='nopad' style='font-size:14px;'>
      The faucet is currently experiencing unusually high mining activity (>{hashrate}). For higher rewards, I recommend visiting again when the activity is lower.
    </span>
    <style>.faucet-status-alert:has(.nopad) { padding: 8px; } </style>
  level: "warn"
  ishtml: true
  filter:
    gt_hashrate: 150000

- key: passport
  prio: 6
  text: |
    <span class='nopad' style='font-size:14px;'>
      New sessions require a minimum gitcoin passport score of at least 2.<br>
      Please visit <a href="https://passport.gitcoin.co/#/" target="_blank">Gitcoin Passport</a> to increase your passport score.
    </span>
    <style>.faucet-status-alert:has(.nopad) { padding: 8px; } </style>
  ishtml: true
  level: info
```

**Comment by pk910** [comment]
> it would be usefully to update generated default config as well
 
 the default config should be updated as well: 
 https://github.com/pk910/PoWFaucet/blob/master/faucet-config.example.yaml#L128
 
 just tested with the command you've mentioned, it generates the new default config with turnstile comment ;)
 note, that the config is not re-generated if it already exists in your data dir.
 
 I'll gonna close this issue as turnstile support is now included & also available since release `v2.2.5`
